gtfslib/converter.py

```diff
@@ -470,6 +451,25 @@ def import_stop(stop, stoptype, zone_ids, item_ids, station_ids=None):
     
     logger.info("Imported %d trips" % n_trips)
 
+    logger.info("Importing transfers...")
+    n_transfers = 0
+    for transfer in gtfs.transfers():
+        from_stop_id = transfer.get('from_stop_id')
+        to_stop_id = transfer.get('to_stop_id')
+        transfer['transfer_type'] = _toint(transfer.get('transfer_type'), 0)
+        for stop_id in (from_stop_id, to_stop_id):
+            if stop_id not in station_ids and stop_id not in stop_ids:
```

Comment:
Maybe we should also do the same test for from/to trip IDs, to test if they exists, and skip the failing transfer in lenient mode or raise an error in strict mode.

Reply:
I added these checks